src/test/components/DragAndDrop.test.js

```diff
@@ -59,4 +56,29 @@ describe('app/DragAndDrop', () => {
     fireEvent.dragExit(dragAndDrop);
     expect(overlay.classList).not.toContain('dragging');
   });
+
+  it('receives profile on file drop', async () => {
+    const { container } = render(
+      <Provider store={createStore()}>
+        <DragAndDrop>Target area here</DragAndDrop>
+      </Provider>
+    );
+    const [dragAndDrop] = container.children;
+
+    const profile = serializeProfile(_getSimpleProfile());
+    const file = new File([profile], 'profile.json', {
+      type: 'application/json',
+    });
+    const files = [file];
+
+    fireEvent.drop(dragAndDrop, { dataTransfer: { files } });
+    await waitFor(() => expect('PROFILE_LOADED').toBeTruthy());
```

Comment:
In this line, note that what you put inside `expect` never change, it's a constant :-) So this is always true.
Instead you should do something like `await waitFor(() => expect(getView(getState()).phase).toBe('DATA_LOADED'))`.

I did the final changes to make the test pass, thanks for the contribution!

Reply:
Thank you, Julien!